dev/langtool/meta/ckeditor.plugin-imagebase/meta.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+# For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+
+captionPlaceholder = Caption displayed below images, which describes it.
```

Comment:
I don't think this description is accurate. It does not describe the image, but prompts user to provide a caption.